Eatery/Controllers/BRB/BRBAccountManager.swift

```diff
@@ -0,0 +1,263 @@
+//
+//  AccountManager.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 10/24/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+struct HistoryEntry {
+    var description = String()
+    var timestamp = String()
+}
+
+enum Stage {
+    case loginScreen
+    case transition
+    case loginFailed
+    case finished(sessionId: String)
+}
+
+// MARK: - Account Settings
+
+typealias LoginInfo = (netid: String, password: String)
+
+private enum BRBAccountSettings {
+    
```

Comment:
Swift seems to favor the singleton pattern over a bunch of static methods. That would mean making the methods not static (i.e. instance), making the initializer private, and adding a `static let BRBAccountSettings shared = BRBAccountSettings()` to the top.